certbot/cli.py

```diff
@@ -1213,8 +1213,16 @@ def prepare_and_parse_args(plugins, args, detect_defaults=False):  # pylint: dis
         " run if an attempt was made to obtain/renew a certificate. If"
         " multiple renewed certificates have identical post-hooks, only"
         " one will be run.")
-    helpful.add("renew", "--renew-hook",
-                action=_RenewHookAction, help=argparse.SUPPRESS)
+    helpful.add(
+        "renew", "--renew-hook",
+        action=_RenewHookAction, help=argparse.SUPPRESS)
+    helpful.add(
+        "renew", "--no-random-sleep-on-renew", action="store_false",
```

Comment:
Let's suppress this instead of displaying it. It's meant for our own tests and for our packagers; we don't want people using it in their cron jobs instead of, say, using hooks. Even worse, we don't want people putting it in tutorials and it getting copied and pasted around, removing the benefit of the random sleep on LE's servers in the first place.

Reply:
Is it the consensus or your opinion? Because it was my opinion also, to make a invisible flag, but @bmw was more on a visible flag in the issue mentionned, and @schoen is concerned about the possibility to disable the logic at all, iirc.

Reply:
More on that on the general discussion.

Reply:
All right, then we all agree! I modify my code accordingly.